services/instagram_service/controller/instagram_controller.py

```diff
@@ -0,0 +1,26 @@
+import asyncio
+from services.instagram_service.src import client_grpc
+from services.instagram_service.src import InstagramService
+class instagram_Contoller:
```

Comment:
Follow Python's PEP 8 style guide for class names, which suggests using CamelCase for class names

Reply:
Please ensure to add two new lines before class definitions as per PEP 8 style guide for better readability and consistency in the code